src/lib/hooks/useTokenList/utils.ts

```diff
@@ -1,21 +1,21 @@
 import { TokenInfo, TokenList } from '@uniswap/token-lists'
 import { WrappedTokenInfo } from 'state/lists/wrappedTokenInfo'
 
-type TokenMap = Readonly<{ [tokenAddress: string]: { token: WrappedTokenInfo; list?: TokenList } }>
-export type ChainTokenMap = Readonly<{ [chainId: number]: TokenMap }>
+type TokenInfoMap = Readonly<{ [tokenAddress: string]: { token: WrappedTokenInfo; list?: TokenList } | undefined }>
+export type ChainToTokenInfoMap = Readonly<{ [chainId: number]: TokenInfoMap }>
```

Comment:
Could this typing be `SupportedChainId` to `TokenInfoMap`, instead of the looser `number`?

Reply:
Will make TokenInfoMap undefineable (is there a good word equivalent to nullable for undefined...?). 

Did you mean to suggest both the optional `?` and the `| undefined`?

Reply:
And yes, will change to ChainTokenInfoMap. I was trying to differentiate it somehow since it's a map to a map... because technically the fully correct version is ChainTokenInfoMapMap... but that feels worse.